### ISSUE TITLE
Cannot read property 'prototype' of undefined when instantiating Axios class from template

### ISSUE THREAD

**Issue by ghost** [acacode/swagger-typescript-api#463]
Hi everyone, how are you doing? I've been playing with the axios template but can't figure out what's wrong here. Whenever I try to instantiate the api I get the following error:

```Cannot read property 'prototype' of undefined```

with BFF being the class exported by the typescript file generated when running the command to generate axios code from swagger docs:

```npx swagger-typescript-api -p ./src/clients/bff/bff.json -o ./src -n clients/bff/bff.ts --axios --api-class-name BFF```

What am I missing here?
<img width="574" alt="Screenshot 2022-11-22 at 19 00 34" src="https://user-images.githubusercontent.com/67009547/203481766-043094c2-c190-46a1-8fe0-5c0d20a1920c.png">
<img width="1077" alt="Screenshot 2022-11-22 at 19 00 23" src="https://user-images.githubusercontent.com/67009547/203481770-8f71e700-c396-461a-9439-d01334ec7f17.png">
<img width="1091" alt="Screenshot 2022-11-22 at 18 59 13" src="https://user-images.githubusercontent.com/67009547/203481772-ac93d4b5-1edc-4480-94b0-8f281e398e97.png">

**Comment by danielcspaivatc** [comment]
Thanks @js2me ! Could you reproduce the problem already? Please let me know if I can help with further information

**Comment by danielcspaivatc** [comment]
Update: I've tried the same Api.ts file in our Next.js project and it worked fine. The problem only occurred in our React Native project.

**Comment by js2me** [comment]
Yes I known about this problem, `api-class-name` not working correctly, but I think I've fixed that in `next`.   
Currently `next` branch has a lot of critical changes and I need more time for create next release

**Comment by js2me** [comment]
can not reproduce, need repro repo
